elasticsearch/ESRESTConnection.js

```diff
@@ -98,20 +98,22 @@ export default class ESRESTConnection extends NUObject {
     }
 
     getESColumns(data, parentKey = null, columns = []) {
+        let isNestedColumn = false;
         if (parentKey !== null && data[0] !== "nested") {
             data = data[0];
         } else if (data[0] === "nested") {
             data = data[1];
+            isNestedColumn = true
         }
 
         Object.entries(data).forEach(([key, val]) => {
             if (objectPath.has(val, 'properties')) {
                 this.getESColumns(Object.values(val), parentKey ? `${parentKey}.${key}` : key, columns);
             } else {
                 if (parentKey === null) {
-                    columns.push(key);
+                    columns.push({key : key, nested : isNestedColumn });
                 } else {
-                    columns.push(`${parentKey}.${key}`);
+                    columns.push({key: `${parentKey}.${key}`, nested : isNestedColumn});
```

Comment:
Isn't this nested, should be nested be true in this case?

Reply:
If the type is nested then it is true. In other cases it is false.